web/react/utils/async_client.jsx

```diff
@@ -521,18 +521,23 @@ export function getPosts(id) {
         return;
     }
 
-    if (PostStore.getAllPosts(channelId) == null) {
+    if ($.isEmptyObject(PostStore.getAllPosts(channelId))) {
         getPostsPage(channelId, Constants.POST_CHUNK_SIZE);
         return;
     }
 
-    const latestUpdate = PostStore.getLatestUpdate(channelId);
+    const latestPost = PostStore.getLatestPost(channelId);
+    let latestPostTime = 0;
+
+    if (latestPost != null && latestPost.update_at != null) {
+        latestPostTime = latestPost.update_at;
```

Comment:
For the order it's create_at but for this we just want posts newer than the latest update since we should already have everything before that update time. If we wanted to get even stricter on what we want then it wouldn't be the latest post's update_at time but the latest update_at time of any post.

We could change it to create_at but we might get a couple extra posts returned that we already have. Either way I'm 0/5